public/app/screens/main/main-screen.jsx

```diff
@@ -7,12 +7,21 @@ var Navigation = require('common/regions/navigation');
 var style = require("./style.scss");
 
 var MainScreen = React.createClass({
+    
+    getInitialState: function() {
+        return {
+            navItems: [
```

Comment:
It need's to be in sync with routes, I would suggest to have such config in routes file.